lib/at_cmd_parser/at_cmd_parser.c

```diff
@@ -251,6 +268,8 @@ static int at_parse_param(const char **at_params_str,
 
 	reset_state();
 
+	setTypeString = false;
```

Comment:
Could ``setTypeString = false;`` simply be inside of the ``reset_state()`` function?

Reply:
Done